tests/multi_ptr/multi_ptr_access_members.h

```diff
@@ -0,0 +1,229 @@
+
+/*******************************************************************************
```

Comment:
```suggestion
/*******************************************************************************
```

Reply:
I've applied this comment [here](https://github.com/KhronosGroup/SYCL-CTS/pull/346/commits/fbc3f591781e925f1893a0630ff8ed49d0817048#diff-7c3b5239659a41d2cc6341067d6094f03525239f4a9380fce5a1056b6bfddc62L1)